python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -27,6 +27,7 @@
     "int": "Torch_IntType",
     "int[]": "TorchIntListType",
     "int?": "TorchOptionalIntType",
+    "int[]?": "TorchOptionalIntListType",
```

Comment:
Can you fix this in the other PR I mentioned before? https://github.com/llvm/torch-mlir/pull/643

Reply:
Do you mind if I change it in this PR?